go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/golang-jwt/jwt
+
+go 1.16
```

Comment:
This should probably be set to 1.14 to be consistent with the matrix build, possibly even lower

Reply:
done